Common/TableProducer/trackextension.cxx

```diff
@@ -58,7 +58,7 @@ struct TrackExtension {
   Service<o2::ccdb::BasicCCDBManager> ccdb;
 
   int mRunNumber;
-  float mMagField;
+  int mMagField;
```

Comment:
If you are going to convert the mag field to float in any case, I am not sure this is a good idea.

Reply:
You are right!
We will get it as an `int` but if we store it as such we will need the conversion on a per particle basis!
I will store it as a float then
Thanks!!